presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -37,9 +38,11 @@
 public class TestAccumuloDistributedQueries
         extends AbstractTestDistributedQueries
 {
-    public TestAccumuloDistributedQueries()
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
     {
-        super(() -> createAccumuloQueryRunner(ImmutableMap.of()));
```

Comment:
how is this allocating resoruces? It was just supplier

Reply:
eg https://github.com/prestosql/presto/pull/2317/files#diff-74b0ca15b398aa39231e786acfe35b5fL38 was starting a mysql server